__tests__/endpoints.test.js

```diff
@@ -114,4 +109,31 @@ describe('GET /api/reviews', () => {
             })
             //Assert
         });
+});
+describe('GET /api/reviews/:review_id/comments', () => {
+    test('Should respond with array of comments for the given review_id', () => {
+        return request(app).get('/api/reviews/3/comments')
+        .expect(200)
+        .then((result) => {
+            result.body.comments.forEach(comment => {
```

Comment:
You should check the length of the comment array as you're using a forEach - you know how many comments there are.

Also, you can check the ordering of the comments too - sorted_by created_at.